apps/dashboard/src/components/billing/plans-row.tsx

```diff
@@ -1,160 +1,231 @@
+import React from 'react';
 import { Badge } from '@/components/primitives/badge';
 import { Card } from '@/components/primitives/card';
 import { Check } from 'lucide-react';
 import { ContactSalesButton } from './contact-sales-button';
 import { PlanActionButton } from './plan-action-button';
+import {
+  ApiServiceLevelEnum,
+  FeatureFlags,
+  FeatureFlagsKeysEnum,
+  FeatureNameEnum,
+  getFeatureForTierAsNumber,
+  getFeatureForTierAsText,
+  StripeBillingIntervalEnum,
+} from '@novu/shared';
 
 interface PlansRowProps {
-  selectedBillingInterval: 'month' | 'year';
-  currentPlan?: 'free' | 'business' | 'enterprise';
+  selectedBillingInterval: StripeBillingIntervalEnum;
+  currentPlan?: ApiServiceLevelEnum;
   trial?: {
     isActive: boolean;
   };
+  featureFlags: FeatureFlags;
 }
-
-interface PlanDisplayProps {
+interface PlanConfig {
+  name: string;
   price: string;
   subtitle: string;
   events: string;
+  features: string[];
+  actionType?: 'button' | 'contact';
 }
 
-function PlanDisplay({ price, subtitle, events }: PlanDisplayProps) {
-  return (
-    <div className="space-y-1">
-      <div className="flex items-baseline gap-1">
-        <span className="text-3xl font-bold tracking-tight">{price}</span>
-        <span className="text-muted-foreground text-sm font-medium">{subtitle}</span>
-      </div>
-      <span className="text-muted-foreground text-sm">{events}</span>
+const PlanFeature: React.FC<{ text: string }> = ({ text }) => (
+  <li className="flex items-center gap-2 text-sm">
+    <Check className="text-primary h-4 w-4" />
+    <span>{text}</span>
+  </li>
+);
+
+const PlanDisplay: React.FC<{
+  price: string;
+  subtitle: string;
+  events: string;
+  isEnterprise?: boolean;
+}> = ({ price, subtitle, events, isEnterprise = false }) => (
+  <div className="space-y-1">
+    <div className="flex items-baseline gap-1">
+      <span className={`${isEnterprise ? 'text-2xl font-semibold' : 'text-3xl font-bold tracking-tight'}`}>
+        {price}
+      </span>
+      {!isEnterprise && <span className="text-muted-foreground text-sm font-medium">{subtitle}</span>}
     </div>
+    {isEnterprise ? (
+      <span className="text-muted-foreground text-sm">For large-scale operations</span>
```

Comment:
Copywriting suggestion 

```suggestion
      <span className="text-muted-foreground text-sm">For scale</span>
```